src/zenml/model/lazy_load.py

```diff
@@ -28,7 +29,23 @@ class ModelVersionDataLazyLoader(BaseModel):
     model version during runtime time of the step.
     """
 
-    model: Model
+    model_name: str
```

Comment:
I might be wrong here, but won't this break all existing runs in the DB, as they store these objects in the DB as part of the configuration?

Reply:
Added migration